packages/go/analysis/go.mod

```diff
@@ -21,6 +21,8 @@ go 1.23
 require (
 	github.com/RoaringBitmap/roaring v1.9.4
 	github.com/bloodhoundad/azurehound/v2 v2.0.1
+	github.com/go-faker/faker/v4 v4.6.0
+	github.com/rs/xid v1.6.0
```

Comment:
I missed the new dependencies earlier. I don't think this one is necessary, for Domain SIDs, we usually just generate one and use it in the test as a constant. This also comes with the upside of it matching the exact format of a microsoft domain SID